cypress/support/commands.js

```diff
@@ -0,0 +1,9 @@
+Cypress.Commands.add('login', (email, password) => {
+  cy.visit("/auth/login")
+  cy.get('input[id="input-email"]').type(email)
+  cy.get('input[id="input-password"]').type(password)
+  cy.get('span.custom-checkbox').click()
+  cy.get('button').eq(0).click()
+  cy.url({timeout: 10000}).should('eq', 'http://localhost:4200/pages/dashboard')
```

Comment:
use only path without http://localhost:4200/

it will fail if you run tests against another env (dev,stage etc.)